test/benchmarks/utils.py

```diff
@@ -56,18 +55,33 @@ def launch_document_store(document_store: str, n_docs: int = 0):
         launch_weaviate(sleep=30, delete_existing=True)
 
 
-def download_from_url(url: str, target_dir: Union[str, Path]):
+def file_previously_downloaded(url_path: Path, target_dir: Union[str, Path]) -> bool:
+    name_of_file_to_download = os.path.split(url_path)[-1]
+
+    if ".tar" in name_of_file_to_download:
+        extracted_dir = name_of_file_to_download.split(".tar")[0]
+        return os.path.exists(f"{target_dir}/{extracted_dir}")
+
+    else:
+        return os.path.exists(f"{target_dir}/{name_of_file_to_download}")
```

Comment:
If we use `pathlib` we can simplify this method quite a bit.

```suggestion
    if ".tar" in url_path.suffixes:
        return Path(target_dir, url_path.parent).exists()
    return Path(target_dir, url_path.name).exists()
```